dev/NavigationView/TestUI/NavigationViewPage.xaml.cs

```diff
@@ -440,7 +440,7 @@ private void NavView_PaneOpened(NavigationView sender, object args)
 
         private void ChangePaneTitle_Click(object sender, RoutedEventArgs e)
         {
-            NavView.PaneTitle = "";
+            NavView.PaneTitle = (String.IsNullOrEmpty(NavView.PaneTitle) ? "NavView Test" : "");
```

Comment:
Need to validate this doesn't break any tests. I just thought this shouldn't be a one-way-only transition for better manual testing value.